### ISSUE TITLE
pt.Trajectory does not load file lists

### ISSUE THREAD

**Issue by pindakaas42** [Amber-MD/pytraj#1521]
This feature was discussed here before. Quoting from the documentation under https://amber-md.github.io/pytraj/latest/_api/pytraj.html#pytraj.Trajectory says:

>>> # load from filename and topology name
>>> traj = pt.Trajectory(fn, tn)
>>> # load from a list of filenames
>>> traj = pt.Trajectory([fn, fn], tn)

This does not load a list of files. Watching my ram fill up I would say the class loads each trajectory in turn,overwriting the last one in the RAM. 

Version: pytraj-2.0.5-py2.7-linux-x86_64.egg/

**Comment by hainm** [comment]
thanks. It was poorly designed for `Trajectory`. All the loading should be done from `pytraj.load`.

**Comment by pindakaas42** [comment]
why not just use pt.load inside the Trajectory __init__ ? this way you dont need to change the Interfact of Trajectory right?

**Comment by hainm** [comment]
> why not just use pt.load inside the Trajectory init ? this way you dont need to change the Interfact of Trajectory right?

yeah, might be a good idea. I did not care much about this originally because I am expecting that user to use `pytraj.load` to get the `Trajectory`. This class is more convenient if you want to create it from memory, something like `Trajectory(xyz=, top=)`.

I bet there are still many corner cases in pytraj  and I want to thank you to dig them up.

**Comment by hainm** [comment]
Fixed in #1551 

```python
In [9]: pt.Trajectory(['tz2.nc', 'tz2.nc'], 'tz2.parm7')                                                                                            
Out[9]: 
pytraj.Trajectory, 202 frames: 
Size: 0.001007 (GB)
<Topology: 223 atoms, 13 residues, 1 mols, non-PBC>      
```

Thanks @pindakaas42
